Arduino-Bling/RobotBling/RobotBling.ino

```diff
@@ -79,17 +79,18 @@ const rgbColor rgbColor_MAGENTA_DIM(0x330033);
  */
 enum
 {
-  OFF = 0,
```

Comment:
In the future, please make sure to build your code and verify it builds before initiating PR.
Later on in your code (see `switch` statement's `default` section), you have `currentCommand = OFF;` Since `OFF` is no longer defined, your code will not compile. Change:
`currentCommand = BLING_COMMAND_OFF,`